src/app/profile/profilepage/page.tsx

```diff
@@ -0,0 +1,62 @@
+import { ProfileView } from "@/components/ProfileView";
+import { Metadata } from "next";
+
+export const metadata: Metadata = {
```

Comment:
I would change this route to `/profile`. 

The edit profile route should be `/profile/edit`